starter-code/src/App.js

```diff
@@ -1,19 +1,92 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './FoodBox';
+import FormFood from './FormFood';
+import SearchBar from './SearchBar';
+import TodaysFoods from './TodaysFoods';
+
 
 class App extends Component {
+  state = {
+    allFoods: foods,
+    // todaysFoods pour pouvoir avoir un tableau avec nos datas à afficher
+    todaysFoods: [],
+    showForm: false,
+  }
+
+  addFood = newFood => {
+    const copyFoods = [...this.state.allFoods];
+    copyFoods.push(newFood)
+    this.setState({ allFoods: copyFoods });
+
+  }
+
+  displayForm = () => {
+    this.setState({ showForm: !this.state.showForm });
+  }
+
+  handleSearch = (value) => {
+    let filteredFood = foods.filter(food => food.name.toUpperCase().includes(value.toUpperCase()));
+    this.setState({ allFoods: filteredFood });
```

Comment:
for this exercice, you should not directly modify the state food otherwise you loose all your new food when you filter it.